terraform-beyond-layer0/example-3/modules/guestbook_service/Dockerrun.aws.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
i think for this (and perhaps example2), we should place the module in a child directory, or even with the source code for  the guestbook example